README.md

```diff
@@ -108,6 +108,11 @@ to see the overall progress of all translations, and which translations are miss
 
 ## Similar projects
 
+- [Better TlDr](https://github.com/emilyseville7cfg-better-tldr)
+  allows you to write almost the same pages as TlDr but in a standardized way
+  and with rich IDE-like facilities like syntax highlighting, refactorings and
+  linting. It is designed to be a replacement for TlDr project.
```

Comment:
> It is designed to be a replacement for TlDr project.

I would refrain from calling it a replacement. An "alternative" could be fine, or even better saying that it's inspired by or derived from this project, but calling it a replacement makes tldr-pages sound deprecated which is not the case.

Reply:
I've removed the sentence about replacement even it's the true goal of my project. I believe that my project is that TlDr should be instead it is now.

Reply:
What you believe you can put in your README. But in this README it needs to be an _objective_ description of your project, not your goals or opinions on it!

Reply:
Noted.